odoo_office365_contacts==UPDATED/odoo_office365_contacts/__init__.py

```
@@ -0,0 +1,2 @@
+# See LICENSE file for full copyright and licensing details.
+from . import models
```

Ruff (0.11.9)

`.models` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)